### PR TITLE
fix: cleanup integration tests

### PR DESCRIPTION
**Pull request by verbanicm**

### CODE REVIEW

File: .github/workflows/setup-gcloud-it.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
   versioned:
     if: ${{ github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name && github.actor != 'dependabot[bot]' }}
     name: setup-gcloud versioned
-    runs-on: ${{ matrix.operating-system }}
+    runs-on: ${{ matrix.os }}
```
Comment (sethvargo) [inline .github/workflows/setup-gcloud-it.yml:20]: Can you move this below the matrix everywhere?

File: .github/workflows/setup-gcloud-it.yml [hunk 2]
```diff
@@ -396,8 +394,6 @@ jobs:
         uses: ./
         with:
           version: 'latest'
-          service_account_email: ${{ secrets.SETUP_GCLOUD_IT_EMAIL }}
```
Comment (sethvargo) [inline .github/workflows/setup-gcloud-it.yml:399]: Why remove that?

Comment (verbanicm) [reply .github/workflows/setup-gcloud-it.yml:399]: it wasnt being used in the code anywhere

File: .github/workflows/setup-gcloud-it.yml (hunk 1 above)
Comment (verbanicm) [reply .github/workflows/setup-gcloud-it.yml]: updated

**Review by sethvargo — APPROVED** [review]

**Pull request merged**